templates/admin/user.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base_private.html" %}
+
+{% from "components/alert.html" import Alert %}
+{% from "components/delete_confirmation.html" import DeleteConfirmation %}
+{% from "components/icon.html" import Icon %}
+{% from "components/modal.html" import Modal %}
+
+{% block content %}
+  <div class="ccpo-panel-container">
+    <div class='col'>
+      <br>
+      <div class="h2">
+        User: {{ user.displayname }} ( {{ user.dod_id }} )
```

Comment:
```suggestion
        User: {{ user.displayname }} ({{ user.dod_id }})
```

No spaces for the parenthesis. Those are shown to the user. Spaces inside the braces (used for jinja) stay.